Dockerfile

```diff
@@ -5,14 +5,14 @@ FROM python:3.11-slim
 # Allow statements and log messages to immediately appear in the logs
```

Comment:
We should make use of the distroless base images as this is the new Google standard.

Reply:
got it, changing it.

Reply:
done.